LiveSPICEVst/EditorView.xaml.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Circuit;
+using Microsoft.Win32;
+using SharpSoundDevice;
+using System;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace LiveSPICEVst
+{
+    /// <summary>
+    /// Dummy VST host class to use the the UI Test App
```

Comment:
Can this class also go in the MockVst app? Or is it needed as a placeholder or something? If that's the case, can you update the comment to clarify?